discord/guild.py

```diff
@@ -2474,6 +2474,38 @@ async def fetch_roles(self) -> List[Role]:
         data = await self._state.http.get_roles(self.id)
         return [Role(guild=self, state=self._state, data=d) for d in data]
 
+    async def fetch_role(self, role_id: int) -> Role:
+        """|coro|
+
+        Retrieves a :class:`Role` that the guild has.
+
+        .. note::
+
+            This method is an API call. For general usage, consider iterating over :attr:`roles` instead.
+
+        .. versionadded:: 1.3
```

Comment:
I think you forgot to change this :)
```suggestion
        .. versionadded:: 2.0
```